__tests__/utils/reference/getValueByPath.test.js

```diff
@@ -11,7 +11,7 @@
  * and limitations under the License.
  */
 import { expect } from 'chai';
-import resolveReference from '../../../lib/utils/references/resolveReference.js';
+import getValueByPath from '../../../lib/utils/references/getValueByPath.js';
```

Comment:
This needed a rename imo, bit confusing, it doesn't actually resolve the reference, just gives you the value of the token using the token "path"